src/Dfc.CourseDirectory.WebV2/ServiceCollectionExtensions.cs

```diff
@@ -125,6 +130,12 @@ public static class ServiceCollectionExtensions
             services.AddSingleton<IProviderContextProvider, ProviderContextProvider>();
             services.AddSingleton(new LoqateAddressSearch.Options() { Key = configuration["PostCodeSearchSettings:Key"] });
             services.AddSingleton<IAddressSearchService, AddressSearchService>();
+            services.AddTransient<ISignInAction, DfeUserInfoHelper>();
```

Comment:
Suspect this is duplicated: SignInActions are declare in a block by themselves above here (`IUkrlpSyncHelper` and `IUkrlpWcfService` are good here).